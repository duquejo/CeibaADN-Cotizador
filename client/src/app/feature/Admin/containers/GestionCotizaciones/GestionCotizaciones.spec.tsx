import * as React from 'react';
import { ShallowWrapper, shallow } from 'enzyme';
import { GestionCotizaciones } from './index';

describe('GestionCotizaciones Test', () => {
  let componentWrapper: ShallowWrapper;

  it('should match snapshot', () => {
    componentWrapper = shallow( <GestionCotizaciones /> );
    expect(componentWrapper).toMatchSnapshot();
  });
});
