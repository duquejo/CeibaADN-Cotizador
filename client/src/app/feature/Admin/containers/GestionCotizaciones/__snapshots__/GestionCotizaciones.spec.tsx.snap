// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GestionCotizaciones Test should match snapshot 1`] = `
<Fragment>
  <h2>
    BÃºsqueda de cotizaciones por Identificador 
    <span
      aria-label="BÃºsqueda cotizaciones"
      role="img"
      title="BÃºsqueda cotizaciones"
    >
      ðŸ”Ž
    </span>
  </h2>
  <p>
    Realice una bÃºsqueda rÃ¡pida por identificador para conocer los valores de la misma.
  </p>
  <styled.div>
    <styled.div>
      <BuscarCotizacion />
    </styled.div>
  </styled.div>
  <styled.hr />
</Fragment>
`;
